governance/program/src/state/proposal.rs

```diff
@@ -209,7 +226,7 @@ pub struct ProposalV2 {
 impl AccountMaxSize for ProposalV2 {
     fn get_max_size(&self) -> Option<usize> {
         let options_size: usize = self.options.iter().map(|o| o.label.len() + 19).sum();
-        Some(self.name.len() + self.description_link.len() + options_size + 295)
+        Some(self.name.len() + self.description_link.len() + options_size + 296)
```

Comment:
@SebastianBor I would like to check with you if adding the byte for the size that comes of adding the `enum` field within the proposal structure is correct here?

Reply:
Yes, it's because it increases the maximum possible size